perun-core/src/main/java/cz/metacentrum/perun/core/entry/ServicesManagerEntry.java

```diff
@@ -594,7 +595,7 @@ public void removeServiceFromServicesPackage(PerunSession sess, ServicesPackage
 	}
 
 	@Override
-	public void addRequiredAttribute(PerunSession sess, Service service, AttributeDefinition attribute) throws PrivilegeException, AttributeNotExistsException, ServiceNotExistsException, AttributeAlreadyAssignedException {
+	public void addRequiredAttribute(PerunSession sess, Service service, AttributeDefinition attribute) throws PrivilegeException, AttributeNotExistsException, ServiceNotExistsException, AttributeAlreadyAssignedException, ServiceAttributesCannotExtend {
```

Comment:
I think we should add this exception to Javadoc in RPC and mark this commit as a breaking change.

Reply:
done